src/verifier.js

```diff
@@ -78,7 +78,7 @@ class LocalVerifier {
         const payload = { [`${this.options.entity}Id`]: id };
         done(null, entity, payload);
       })
-      .catch(error => error ? done(error) : done(null, error));
+      .catch(error => error ? done(error) : done(null, error, { message: 'Invalid login' }));
```

Comment:
@ekryski Yeah.  I wasn't sure exactly what would happen here: https://github.com/jaredhanson/passport-local/blob/master/lib/strategy.js#L81 where passport does a special check for `!user`, which is probably what you're talking about.  I don't really understand the need to differentiate it.  Why does Passport care?